lib/solaar/ui/window.py

```diff
@@ -305,7 +305,10 @@ def _create_window_layout():
 	panel.pack_start(_empty, True, True, 0)
 
 	bottom_buttons_box = Gtk.ButtonBox(Gtk.Orientation.HORIZONTAL)
-	bottom_buttons_box.set_layout(Gtk.ButtonBoxStyle.START)
+	bottom_buttons_box.set_layout(Gtk.ButtonBoxStyle.SPREAD)
```

Comment:
Keep the old layout, just add some spacing between the buttons.

Reply:
Done.  I added 20 pixels of space as that seemed adequate and not excessive.

Reply:
Looks good, thanks :)